docs/python-versions.md

```diff
@@ -18,14 +18,12 @@ Many Python version request formats are supported:
 
 - `<version>` e.g. `3`, `3.12`, `3.12.3`
 - `<version-specifier>` e.g. `>=3.12,<3.13`
-- `<implementation>` e.g. `cpython` or `cp`
-- `<implementation>@<version>` e.g. `cpython@3.12`
-- `<implementation><version>` e.g. `cpython3.12` or `cp312`
+- `<implementation>` e.g. `cpython` or `cp`, `pypy` or `pp`
+- `<implementation>@<version>` e.g. `cpython@3.12` or `pypy@3.12`
+- `<implementation><version>` e.g. `cpython3.12` or `cp312`, `pypy3.12` or `pp312`
```

Comment:
I wonder if we should be enumerating these all here? Maybe we should start a separate section for supported implementations — this is also missing GraalPy and I don't want to to be too verbose.